index.html

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Tribute Page</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <main class="container" id="main">
+      <!-- Header section-->
+      <header>
+        <h1 id="title">Dr. Norman Borlaug</h1>
+        <p>The man who saved a billion lives</p>
+      </header>
+      <!-- Video Section -->
+      <div class="section" id="img-div">
```

Comment:
You should use the semantic HTML tag, Section, to help improve the search ranking of your website.

Reply:
- [ ] I suggest you use rephrase your comments politely to sound like suggestions instead of orders, you could guide from this example:

>This is a polite suggestion: "I think it would be better to use semantic tags"
>
>This reads as an order: "Use semantic tags"

Please consider editing this comment and applying the suggestion! 🤓 🛠️